config/output_schemas/ni_sas_schema.toml

```diff
@@ -0,0 +1,151 @@
+[ref]
```

Comment:
NI SAS schema is exactly the same as GB SAS schema. Shall we keep just one toml file, "sas_schema.toml" and repeat it two times in developer config:

gb_sas_schema: "config/output_schemas/sas_schema.toml"
ni_sas_schema: "config/output_schemas/sas_schema.toml"

Reply:
Yes - good idea if the fields are going to be the same.  We are waiting for confirmation if all cols are required, as some cols are missing.  I have commented these out in this version.   When we have had the final schema from the team, can we review then?